pkg/routeagent_driver/handlers/ovn/gateway_dataplane.go

```diff
@@ -0,0 +1,139 @@
+package ovn
+
+import (
+	"net"
+	"os"
+
+	"github.com/coreos/go-iptables/iptables"
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+	"k8s.io/klog"
+
+	npSyncerOvn "github.com/submariner-io/submariner/pkg/networkplugin-syncer/handlers/ovn"
+	"github.com/submariner-io/submariner/pkg/routeagent_driver/constants"
+	"github.com/submariner-io/submariner/pkg/util"
+)
+
+func (ovn *Handler) cleanupGatewayDataplane() error {
+	currentRemoteSubnets, err := ovn.getExistingRuleSubnets()
+	if err != nil {
+		return errors.Wrapf(err, "error reading ip rule list for IPv4")
```

Comment:
Trying to get the error message and method name to match more...

```suggestion
	currentRemoteSubnets, err := ovn.getExistingIPv4RuleSubnets()
	if err != nil {
		return errors.Wrapf(err, "error reading IP rule subnets for IPv4")
```

Reply:
Done.